src/platform/ESP32/Logging.cpp

```diff
@@ -26,35 +26,20 @@ void ENFORCE_FORMAT(3, 0) LogV(const char * module, uint8_t category, const char
     snprintf(tag, sizeof(tag), "chip[%s]", module);
     tag[sizeof(tag) - 1] = 0;
 
+    char formattedMsg[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
```

Comment:
Blocking comment to prevent merging PR after approval.

We are working on a forward fix. If that forward fix doesn't materialize I will mark this comment as resolved and we will merge this EOD